packages/opentelemetry-resources/src/index.ts

```diff
@@ -14,22 +14,19 @@
  * limitations under the License.
  */
 
-export { Resource } from './Resource';
-export { IResource } from './IResource';
-export { defaultServiceName } from './platform';
-export { DetectorSync, Detector } from './types';
 export { ResourceDetectionConfig } from './config';
+export { detectResources } from './detect-resources';
 export {
   browserDetector,
   browserDetectorSync,
```

Comment:
Hmm, is there a reason `browserDetectorSync` is still here? 🤔

Reply:
Mass-edit failure